docs/content.zh/release-notes/flink-1.13.md

```diff
@@ -0,0 +1,248 @@
+
+#### Update the Row.toString method
+
+##### [FLINK-18090](https://issues.apache.org/jira/browse/FLINK-18090)
+
+The Row.toSting method has been reworked. This is an incompatible change. If the legacy
+representation is still required for tests, the old behavior can be restored via the flag
+`RowUtils.USE_LEGACY_TO_STRING` for the local JVM. However, relying on the row's string
+representation for tests is not a good idea in general as field data types are not verified.
+
+### Runtime
+
+#### BoundedOneInput.endInput is called when taking synchronous savepoint
+
+##### [FLINK-21132](https://issues.apache.org/jira/browse/FLINK-21132)
+
+`endInput()` is not called anymore (on BoundedOneInput and BoundedMultiInput) when the job is
+stopping with savepoint.
+
+#### Remove JobManagerOptions.SCHEDULING_STRATEGY
+
+##### [FLINK-20591](https://issues.apache.org/jira/browse/FLINK-20591)
+
+The configuration parameter `jobmanager.scheduler.scheduling-strategy` has been removed, because
+the `legacy` scheduler has been removed from Flink 1.13.0.
+
+#### Warn user if System.exit() is called in user code
+
+##### [FLINK-15156](https://issues.apache.org/jira/browse/FLINK-15156)
+
+A new configuration value `cluster.intercept-user-system-exit` allows to log a warning, or throw an
+exception if user code calls `System.exit()`.
+
+This feature is not covering all locations in Flink where user code is executed. It just adds the
+infrastructure for such an interception.
+
+### Docker
+
+#### Consider removing automatic configuration fo number of slots from docker
+
+##### [FLINK-21036](https://issues.apache.org/jira/browse/FLINK-21036)
+
+The docker images no longer set the default number of taskmanager slots to the number of CPU cores.
+This behavior was inconsistent with all other deployment methods and ignored any limits on the CPU
+usage set via docker.
+
+#### Rework jemalloc switch to use an environment variable
+
+##### [FLINK-21034](https://issues.apache.org/jira/browse/FLINK-21034)
+
+The docker switch for disabling the jemalloc memory allocator has been reworked from a script
+argument to an environment variable called DISABLE_JEMALLOC. If set to "true" jemalloc will not be
+enabled.
+
+### Connectors
+
+#### Remove swift FS filesystem
+
+##### [FLINK-21819](https://issues.apache.org/jira/browse/FLINK-21819)
+
+The Swift FileSystem is no longer being actively developed and has been removed from the project and
+distribution.
+
+### Monitoring & debugging
+
+#### Introduce latency tracking state
+
+##### [FLINK-21736](https://issues.apache.org/jira/browse/FLINK-21736)
+
+State access latency metrics are introduced to track all kinds of keyed state access to help debug
+state performance. This feature is not enabled by default and could be turned on once
+setting `state.backend.latency-track.keyed-state-enabled` as true.
+
+#### Support for CPU FlameGraphs in new web UI
```

Comment:
Is there really a whole new web UI? That'd seem more noteworthy than a specific feature within it.

Reply:
The titles are extracted from JIRA issues, I suppose back when the issue was created we had two UIs at the same time.